### ISSUE TITLE
Expecting `AbstractCommand[]` as a typehint.

### ISSUE THREAD

**Issue by ElvenSpellmaker** [ve-interactive/PhpSnifferRules#6]
The sniffer doesn't understand common practises such as the 'typed' array below.

```PHP
	/**
	 * Connects commands together using Inputs and Outputs.
	 *
	 * @param AbstractCommand[] $commands
	 *
	 * @return void
	 */
	public function connect(array $commands);
```

**Comment by ElvenSpellmaker** [comment]
IMO the sniffer should accept and use the `[]` on the end to assume a typehint of `array`.

**Comment by JackBlower** [comment]
Fixed with #7 .
